### DISCUSSION TITLE
Feature Request: Ability to review commit logs

### DISCUSSION THREAD

**Discussion by bsdjhb** [community/community#128672] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I have found that commit logs are often just as important as the diffs in a commit, and there are many instances where the feedback I provide on a review is on the commit log itself, not just the diff.  Specifically, I would like to be able to comment on the commit log including making suggested changes to the commit log (just as you can make suggested changes to the diff).  Currently in a multi-commit series I pick a random changed line to add a comment to when requesting changes to a commit log, but it is not always obvious in the Conversation view which commit the comment is about.  Having the option of making suggestions using the suggestion editor would also be a nice improvement vs manually cutting and pasting the relevant bits of the log into the comment.

**Comment by bsdimp** [comment]
I've often wanted to suggest something better in github pull reviews and have had to fall back to awkward second bests. Whether it's spelling errors, terrible grammar, too much detail, too little, etc. It's best to comment on the thing directly.

It would also allow automated commit message style checkers to comment on what the problem is directly, like lack of SOB line (which now has to be done indirectly).
